packages/in_app_purchase/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.3.4+2
```

Comment:
new line after the version

Reply:
Done

Reply:
The diff seems odd, CHANGELOG probably needs to be re-based